### ISSUE TITLE
How could I download the dataset?

### ISSUE THREAD

**Issue by yoyolin** [rccohn/NEU-Cluster#14]
Hi I am just leaving a comment to see how could I get the dataset and what is the license for the dataset? Thank you!

**Comment by rccohn** [comment]
From README.md:

> The NEU dataset [2] has been open-sourced and is available here: http://faculty.neu.edu.cn/yunhyan/NEU_surface_defect_database.html. (Sometimes the link is broken, so if it doesn't work you reach out I can find a way to get you the images.)

If the site is down let me know and I can figure out a way to access the images for you.

**Comment by yoyolin** [comment]
> From README.md:
> 
> > The NEU dataset [2] has been open-sourced and is available here: http://faculty.neu.edu.cn/yunhyan/NEU_surface_defect_database.html. (Sometimes the link is broken, so if it doesn't work you reach out I can find a way to get you the images.)
> 
> If the site is down let me know and I can figure out a way to access the images for you.

I was not able to open the page to see. That's why I reached out. Sorry for the confusion.

**Comment by rccohn** [comment]
Yeah it looks like the website is currently down. Unfortunately since it's not our dataset we can't host it publicly. However, if you send me an email (rcohn@cmu.edu,) I can send it to you directly.

**Comment by zaidbhat1234** [comment]
I tried sending out an email to rcohn@cmu.edu. However it no longer seems to work. @rccohn Could you please let me know how I can access the dataset?

Thanks!

**Comment by rccohn** [comment]
Yup the punishment for graduating is that my cmu email no longer works... Anyways check out the [readme](https://github.com/rccohn/NEU-Cluster/blob/master/README.md), it has instructions for getting the dataset from kaggle. I just checked the link and it looks like it still works!
